clang warning: (misleading indentation) in bt.c

```
bt.c:301:4: warning: misleading indentation; statement is not part of the previous 'for' [-Wmisleading-indentation]
                        fprintf(stdout, "Unused Slots in Offset Table:%Lf %%\n", 100.00 * (long double)(offset_table_size - i) / (long double)(offset_table_size));
                        ^
bt.c:299:3: note: previous statement is here
                for (i = 0; i < offset_table_size && offset_data[i].collisions; i++)
                ^
1 warning generated.
```
Indeed, there's a `;` before the `fprintf`

--- a/src/bt.c
+++ b/src/bt.c
@@ -112,13 +112,13 @@ void bt_free(void **ptr)
 
 void bt_error_fn(const char *str, char *file, int line)
 {
-      fprintf(stderr, "%s in file:%s, line:%d.\n", str, file, line);
-      error();
+	fprintf(stderr, "%s in file:%s, line:%d.\n", str, file, line);
+	error();
 }
 
 void bt_warn_fn(const char *str, char *file, int line)
 {
-      fprintf(stderr, "%s in file:%s, line:%d.\n", str, file, line);
+	fprintf(stderr, "%s in file:%s, line:%d.\n", str, file, line);
 }
 
 static unsigned int modulo_op(void * hash, unsigned int N, uint64_t shift64, uint64_t shift128)
@@ -222,64 +222,64 @@ static void init_tables(unsigned int approx_offset_table_sz, unsigned int approx
 #if _OPENMP
 #pragma omp parallel private(i, offset_data_idx)
 #endif
-{
+	{
 #if _OPENMP
 #pragma omp for
 #endif
-	for (i = 0; i < offset_table_size; i++) {
-		//memset(&offset_data[i], 0, sizeof(auxilliary_offset_data));
-		offset_data[i].offset_table_idx = 0;
-		offset_data[i].collisions = 0;
-		offset_data[i].hash_location_list = NULL;
-		offset_data[i].iter = 0;
-		offset_table[i] = 0;
-	}
+		for (i = 0; i < offset_table_size; i++) {
+			//memset(&offset_data[i], 0, sizeof(auxilliary_offset_data));
+			offset_data[i].offset_table_idx = 0;
+			offset_data[i].collisions = 0;
+			offset_data[i].hash_location_list = NULL;
+			offset_data[i].iter = 0;
+			offset_table[i] = 0;
+		}
 #if _OPENMP
 #pragma omp barrier
 #endif
-	/* Build Auxiliary data structure for offset_table. */
+		/* Build Auxiliary data structure for offset_table. */
 #if _OPENMP
 #pragma omp for
 #endif
-	for (i = 0; i < num_loaded_hashes; i++) {
-		offset_data_idx = modulo_op(loaded_hashes + i * binary_size_actual, offset_table_size, shift64_ot_sz, shift128_ot_sz);
+		for (i = 0; i < num_loaded_hashes; i++) {
+			offset_data_idx = modulo_op(loaded_hashes + i * binary_size_actual, offset_table_size, shift64_ot_sz, shift128_ot_sz);
 #if _OPENMP
 #pragma omp atomic
 #endif
-		offset_data[offset_data_idx].collisions++;
-	}
+			offset_data[offset_data_idx].collisions++;
+		}
 #if _OPENMP
 #pragma omp barrier
 #pragma omp single
 #endif
-	for (i = 0; i < offset_table_size; i++)
-	      if (offset_data[i].collisions) {
-			if (bt_malloc((void **)&offset_data[i].hash_location_list, offset_data[i].collisions * sizeof(unsigned int)))
-				bt_error("Failed to allocate memory: offset_data[i].hash_location_list.");
-			if (offset_data[i].collisions > max_collisions)
-				max_collisions = offset_data[i].collisions;
-	      }
+		for (i = 0; i < offset_table_size; i++)
+			if (offset_data[i].collisions) {
+				if (bt_malloc((void **)&offset_data[i].hash_location_list, offset_data[i].collisions * sizeof(unsigned int)))
+					bt_error("Failed to allocate memory: offset_data[i].hash_location_list.");
+				if (offset_data[i].collisions > max_collisions)
+					max_collisions = offset_data[i].collisions;
+			}
 #if _OPENMP
 #pragma omp barrier
 MAYBE_PARALLEL_FOR
 #endif
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int iter;
-		offset_data_idx = modulo_op(loaded_hashes + i * binary_size_actual, offset_table_size, shift64_ot_sz, shift128_ot_sz);
+			for (i = 0; i < num_loaded_hashes; i++) {
+				unsigned int iter;
+				offset_data_idx = modulo_op(loaded_hashes + i * binary_size_actual, offset_table_size, shift64_ot_sz, shift128_ot_sz);
 #if _OPENMP
 MAYBE_ATOMIC_WRITE
 #endif
-		offset_data[offset_data_idx].offset_table_idx = offset_data_idx;
+					offset_data[offset_data_idx].offset_table_idx = offset_data_idx;
 #if _OPENMP
 MAYBE_ATOMIC_CAPTURE
 #endif
-		iter = offset_data[offset_data_idx].iter++;
-		offset_data[offset_data_idx].hash_location_list[iter] = i;
-	}
+					iter = offset_data[offset_data_idx].iter++;
+				offset_data[offset_data_idx].hash_location_list[iter] = i;
+			}
 #if _OPENMP
 #pragma omp barrier
 #endif
-}
+	}
 	total_memory_in_bytes += num_loaded_hashes * sizeof(unsigned int);
 
 	//qsort((void *)offset_data, offset_table_size, sizeof(auxilliary_offset_data), qsort_compare);
@@ -298,7 +298,7 @@ MAYBE_ATOMIC_CAPTURE
 
 		for (i = 0; i < offset_table_size && offset_data[i].collisions; i++)
 			;
-			fprintf(stdout, "Unused Slots in Offset Table:%Lf %%\n", 100.00 * (long double)(offset_table_size - i) / (long double)(offset_table_size));
+		fprintf(stdout, "Unused Slots in Offset Table:%Lf %%\n", 100.00 * (long double)(offset_table_size - i) / (long double)(offset_table_size));
 
 		fprintf(stdout, "Total Memory Use(in GBs):%Lf\n", ((long double)total_memory_in_bytes) / ((long double) 1024 * 1024 * 1024));
 	}
@@ -425,13 +425,13 @@ static unsigned int create_tables()
 				j = 1;
 				while (j <= backtrack_steps && (int)(i - j) >= 0) {
 					last_offset = offset_table[offset_data[i - j].offset_table_idx];
-					 iter = 0;
+					iter = 0;
 					while (iter < offset_data[i - j].collisions) {
 						hash_table_idx =
 							calc_ht_idx(offset_data[i - j].hash_location_list[iter],
-								    last_offset);
-							assign0_ht(hash_table_idx);
-							iter++;
+							            last_offset);
+						assign0_ht(hash_table_idx);
+						iter++;
 					}
 					offset_table[offset_data[i - j].offset_table_idx] = 0;
 					done -= offset_data[i - j].collisions;
@@ -537,11 +537,11 @@ static unsigned int next_prime(unsigned int num)
 }
 
 unsigned int create_perfect_hash_table(int htype, void *loaded_hashes_ptr,
-			       unsigned int num_ld_hashes,
-			       OFFSET_TABLE_WORD **offset_table_ptr,
-			       unsigned int *offset_table_sz_ptr,
-			       unsigned int *hash_table_sz_ptr,
-			       unsigned int verb)
+                                       unsigned int num_ld_hashes,
+                                       OFFSET_TABLE_WORD **offset_table_ptr,
+                                       unsigned int *offset_table_sz_ptr,
+                                       unsigned int *hash_table_sz_ptr,
+                                       unsigned int verb)
 {
 	long double multiplier_ht, multiplier_ot, inc_ht, inc_ot;
 	unsigned int approx_hash_table_sz, approx_offset_table_sz, i, dupe_remove_ht_sz;

--- a/src/bt_hash_type_128.c
+++ b/src/bt_hash_type_128.c
@@ -120,10 +120,10 @@ int test_tables_128(unsigned int num_loaded_hashes, OFFSET_TABLE_WORD *offset_ta
 			hash_table_collisions[hash_table_idx]++;
 
 			if (error && (hash_table_128[hash_table_idx] != (unsigned int)(hash.LO64 & 0xffffffff)  ||
-			    hash_table_128[hash_table_idx + hash_table_size] != (unsigned int)(hash.LO64 >> 32) ||
-			    hash_table_128[hash_table_idx + 2 * hash_table_size] != (unsigned int)(hash.HI64 & 0xffffffff) ||
-			    hash_table_128[hash_table_idx + 3 * hash_table_size] != (unsigned int)(hash.HI64 >> 32) ||
-			    hash_table_collisions[hash_table_idx] > 1)) {
+			              hash_table_128[hash_table_idx + hash_table_size] != (unsigned int)(hash.LO64 >> 32) ||
+			              hash_table_128[hash_table_idx + 2 * hash_table_size] != (unsigned int)(hash.HI64 & 0xffffffff) ||
+			              hash_table_128[hash_table_idx + 3 * hash_table_size] != (unsigned int)(hash.HI64 >> 32) ||
+			              hash_table_collisions[hash_table_idx] > 1)) {
 				fprintf(stderr, "Error building tables: Loaded hash idx:%u, No. of collisions:%u\n", i, hash_table_collisions[hash_table_idx]);
 				error = 0;
 			}
@@ -198,9 +198,9 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 
 	counter = 0;
 	for (i = 0; i < hash_table_size; i++) {
-		 hash_table[i].collisions = collisions[i];
-		 hash_table[i].iter = 0;
-		 hash_table[i].store_loc1 = hash_table[i].store_loc2 =
+		hash_table[i].collisions = collisions[i];
+		hash_table[i].iter = 0;
+		hash_table[i].store_loc1 = hash_table[i].store_loc2 =
 			hash_table[i].idx_hash_loc_list = 0xffffffff;
 		if (hash_table[i].collisions > 3)
 			hash_table[i].idx_hash_loc_list = counter++;
@@ -211,11 +211,11 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 
 	counter = 0;
 	for (i = 0; i < hash_table_size; i++)
-	      if (collisions[i] > 3) {
+		if (collisions[i] > 3) {
 			if (bt_malloc((void **)&hash_location_list[counter], (collisions[i] - 1) * sizeof(unsigned int)))
 				bt_error("Failed to allocate memory: hash_location_list[counter].");
 			counter++;
-	      }
+		}
 
 	for (i = 0; i < num_loaded_hashes; i++) {
 		unsigned int k = rehash_list[i];
@@ -242,7 +242,7 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 					hash_table[idx].store_loc2 = k;
 			}
 			else if (check_equal(hash_table[idx].store_loc1, k) ||
-				 check_equal(hash_table[idx].store_loc2, k))
+			         check_equal(hash_table[idx].store_loc2, k))
 				set_zero(k);
 		}
 
@@ -302,125 +302,125 @@ unsigned int remove_duplicates_128(unsigned int num_loaded_hashes, unsigned int 
 #if _OPENMP
 #pragma omp parallel private(i)
 #endif
-{
+	{
 #if _OPENMP
 #pragma omp for
 #endif
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_128[i].LO64 & (hash_table_size - 1);
+		for (i = 0; i < num_loaded_hashes; i++) {
+			unsigned int idx = loaded_hashes_128[i].LO64 & (hash_table_size - 1);
 #if _OPENMP
 #pragma omp atomic
 #endif
-		collisions[idx]++;
-	}
+			collisions[idx]++;
+		}
 
-	counter = 0;
+		counter = 0;
 #if _OPENMP
 #pragma omp barrier
 
 #pragma omp for
 #endif
-	for (i = 0; i < hash_table_size; i++) {
-		hash_table[i].iter = 0;
-		if (collisions[i] > 4)
+		for (i = 0; i < hash_table_size; i++) {
+			hash_table[i].iter = 0;
+			if (collisions[i] > 4)
 #if _OPENMP
 #pragma omp atomic
 #endif
-			 counter += (collisions[i] - 3);
-	}
+				counter += (collisions[i] - 3);
+		}
 #if _OPENMP
 #pragma omp barrier
 
 #pragma omp sections
 #endif
-{
+		{
 #if _OPENMP
 #pragma omp section
 #endif
-{
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_128[i].LO64 & (hash_table_size - 1);
+			{
+				for (i = 0; i < num_loaded_hashes; i++) {
+					unsigned int idx = loaded_hashes_128[i].LO64 & (hash_table_size - 1);
 
-		if (collisions[idx] == 2) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
+					if (collisions[idx] == 2) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (check_equal(hash_table[idx].store_loc1, i))
+							set_zero(i);
+					}
+				}
 			}
-			else if (check_equal(hash_table[idx].store_loc1, i))
-				set_zero(i);
-		}
-	}
-}
 #if _OPENMP
 #pragma omp section
 #endif
-{
-	if (bt_malloc((void **)&rehash_list, counter * sizeof(unsigned int)))
-		bt_error("Failed to allocate memory: rehash_list.");
-	counter = 0;
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_128[i].LO64 & (hash_table_size - 1);
+			{
+				if (bt_malloc((void **)&rehash_list, counter * sizeof(unsigned int)))
+					bt_error("Failed to allocate memory: rehash_list.");
+				counter = 0;
+				for (i = 0; i < num_loaded_hashes; i++) {
+					unsigned int idx = loaded_hashes_128[i].LO64 & (hash_table_size - 1);
 
-		if (collisions[idx] == 3) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
-			}
-			else if (hash_table[idx].iter == 1) {
-				if (check_equal(hash_table[idx].store_loc1, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc2 = i;
-				}
-			}
-			else if (check_equal(hash_table[idx].store_loc1, i) ||
-				 check_equal(hash_table[idx].store_loc2, i))
-				set_zero(i);
-		}
+					if (collisions[idx] == 3) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (hash_table[idx].iter == 1) {
+							if (check_equal(hash_table[idx].store_loc1, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc2 = i;
+							}
+						}
+						else if (check_equal(hash_table[idx].store_loc1, i) ||
+						         check_equal(hash_table[idx].store_loc2, i))
+							set_zero(i);
+					}
 
-		else if (collisions[idx] >= 4) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
-			}
-			else if (hash_table[idx].iter == 1) {
-				if (check_equal(hash_table[idx].store_loc1, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc2 = i;
+					else if (collisions[idx] >= 4) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (hash_table[idx].iter == 1) {
+							if (check_equal(hash_table[idx].store_loc1, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc2 = i;
+							}
+
+						}
+						else if (hash_table[idx].iter == 2) {
+							if (check_equal(hash_table[idx].store_loc1, i) ||
+							    check_equal(hash_table[idx].store_loc2, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc3 = i;
+							}
+						}
+						else if (hash_table[idx].iter >= 3) {
+							if (check_equal(hash_table[idx].store_loc1, i) ||
+							    check_equal(hash_table[idx].store_loc2, i) ||
+							    check_equal(hash_table[idx].store_loc3, i))
+								set_zero(i);
+							else {
+								if (collisions[idx] > 4)
+									rehash_list[counter++] = i;
+							}
+						}
+					}
 				}
 
-			}
-			else if (hash_table[idx].iter == 2) {
-				if (check_equal(hash_table[idx].store_loc1, i) ||
-				    check_equal(hash_table[idx].store_loc2, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc3 = i;
-				}
-			}
-			else if (hash_table[idx].iter >= 3) {
-				if (check_equal(hash_table[idx].store_loc1, i) ||
-				    check_equal(hash_table[idx].store_loc2, i) ||
-				    check_equal(hash_table[idx].store_loc3, i))
-					set_zero(i);
-				else {
-					if (collisions[idx] > 4)
-						rehash_list[counter++] = i;
-				}
+				if (counter)
+					remove_duplicates_final(counter, counter + (counter >> 1), rehash_list);
+				bt_free((void **)&rehash_list);
 			}
 		}
 	}
-
-	if (counter)
-		remove_duplicates_final(counter, counter + (counter >> 1), rehash_list);
-	bt_free((void **)&rehash_list);
-}
-}
-}
 
 #if 0
 	{	unsigned int col1 = 0, col2 = 0, col3 = 0, col4 = 0, col5a = 0;
@@ -440,8 +440,8 @@ unsigned int remove_duplicates_128(unsigned int num_loaded_hashes, unsigned int 
 		col3 *= 3;
 		col4 *= 4;
 		fprintf(stderr, "Statistics:%Lf %Lf %Lf %Lf %Lf\n", (long double)col1 / (long double)num_loaded_hashes,
-		  (long double)col2 / (long double)num_loaded_hashes, (long double)col3 / (long double)num_loaded_hashes,
-			(long double)col4 / (long double)num_loaded_hashes, (long double)col5a / (long double)num_loaded_hashes);
+		        (long double)col2 / (long double)num_loaded_hashes, (long double)col3 / (long double)num_loaded_hashes,
+		        (long double)col4 / (long double)num_loaded_hashes, (long double)col5a / (long double)num_loaded_hashes);
 
 	}
 #endif

--- a/src/bt_hash_type_192.c
+++ b/src/bt_hash_type_192.c
@@ -124,12 +124,12 @@ int test_tables_192(unsigned int num_loaded_hashes, OFFSET_TABLE_WORD *offset_ta
 			hash_table_collisions[hash_table_idx]++;
 
 			if (error && (hash_table_192[hash_table_idx] != (unsigned int)(hash.LO & 0xffffffff) ||
-				hash_table_192[hash_table_idx + hash_table_size] != (unsigned int)(hash.LO >> 32) ||
-				hash_table_192[hash_table_idx + 2 * hash_table_size] != (unsigned int)(hash.MI & 0xffffffff) ||
-				hash_table_192[hash_table_idx + 3 * hash_table_size] != (unsigned int)(hash.MI >> 32) ||
-				hash_table_192[hash_table_idx + 4 * hash_table_size] != (unsigned int)(hash.HI & 0xffffffff) ||
-				hash_table_192[hash_table_idx + 5 * hash_table_size] != (unsigned int)(hash.HI >> 32) ||
-				hash_table_collisions[hash_table_idx] > 1)) {
+			              hash_table_192[hash_table_idx + hash_table_size] != (unsigned int)(hash.LO >> 32) ||
+			              hash_table_192[hash_table_idx + 2 * hash_table_size] != (unsigned int)(hash.MI & 0xffffffff) ||
+			              hash_table_192[hash_table_idx + 3 * hash_table_size] != (unsigned int)(hash.MI >> 32) ||
+			              hash_table_192[hash_table_idx + 4 * hash_table_size] != (unsigned int)(hash.HI & 0xffffffff) ||
+			              hash_table_192[hash_table_idx + 5 * hash_table_size] != (unsigned int)(hash.HI >> 32) ||
+			              hash_table_collisions[hash_table_idx] > 1)) {
 				fprintf(stderr, "Error building tables: Loaded hash idx:%u, No. of collisions:%u\n", i, hash_table_collisions[hash_table_idx]);
 				error = 0;
 			}
@@ -201,9 +201,9 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 
 	counter = 0;
 	for (i = 0; i < hash_table_size; i++) {
-		 hash_table[i].collisions = collisions[i];
-		 hash_table[i].iter = 0;
-		 hash_table[i].store_loc1 = hash_table[i].store_loc2 =
+		hash_table[i].collisions = collisions[i];
+		hash_table[i].iter = 0;
+		hash_table[i].store_loc1 = hash_table[i].store_loc2 =
 			hash_table[i].idx_hash_loc_list = 0xffffffff;
 		if (hash_table[i].collisions > 3)
 			hash_table[i].idx_hash_loc_list = counter++;
@@ -214,11 +214,11 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 
 	counter = 0;
 	for (i = 0; i < hash_table_size; i++)
-	      if (collisions[i] > 3) {
+		if (collisions[i] > 3) {
 			if (bt_malloc((void **)&hash_location_list[counter], (collisions[i] - 1) * sizeof(unsigned int)))
 				bt_error("Failed to allocate memory: hash_location_list[counter].");
 			counter++;
-	      }
+		}
 
 	for (i = 0; i < num_loaded_hashes; i++) {
 		unsigned int k = rehash_list[i];
@@ -245,7 +245,7 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 					hash_table[idx].store_loc2 = k;
 			}
 			else if (check_equal(hash_table[idx].store_loc1, k) ||
-				 check_equal(hash_table[idx].store_loc2, k))
+			         check_equal(hash_table[idx].store_loc2, k))
 				set_zero(k);
 		}
 
@@ -305,126 +305,126 @@ unsigned int remove_duplicates_192(unsigned int num_loaded_hashes, unsigned int 
 #if _OPENMP
 #pragma omp parallel private(i)
 #endif
-{
+	{
 #if _OPENMP
 #pragma omp for
 #endif
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_192[i].LO & (hash_table_size - 1);
+		for (i = 0; i < num_loaded_hashes; i++) {
+			unsigned int idx = loaded_hashes_192[i].LO & (hash_table_size - 1);
 #if _OPENMP
 #pragma omp atomic
 #endif
-		collisions[idx]++;
-	}
+			collisions[idx]++;
+		}
 
-	counter = 0;
+		counter = 0;
 #if _OPENMP
 #pragma omp barrier
 
 #pragma omp for
 #endif
-	for (i = 0; i < hash_table_size; i++) {
-		  hash_table[i].iter = 0;
-		 if (collisions[i] > 4)
+		for (i = 0; i < hash_table_size; i++) {
+			hash_table[i].iter = 0;
+			if (collisions[i] > 4)
 #if _OPENMP
 #pragma omp atomic
 #endif
-			 counter += (collisions[i] - 3);
-	}
+				counter += (collisions[i] - 3);
+		}
 #if _OPENMP
 #pragma omp barrier
 
 #pragma omp sections
 #endif
-{
+		{
 #if _OPENMP
 #pragma omp section
 #endif
-{
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_192[i].LO & (hash_table_size - 1);
+			{
+				for (i = 0; i < num_loaded_hashes; i++) {
+					unsigned int idx = loaded_hashes_192[i].LO & (hash_table_size - 1);
 
-		if (collisions[idx] == 2) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
+					if (collisions[idx] == 2) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (check_equal(hash_table[idx].store_loc1, i))
+							set_zero(i);
+					}
+				}
 			}
-			else if (check_equal(hash_table[idx].store_loc1, i))
-				set_zero(i);
-		}
-	}
-}
 
 #if _OPENMP
 #pragma omp section
 #endif
-{
-	if (bt_malloc((void **)&rehash_list, counter * sizeof(unsigned int)))
-		bt_error("Failed to allocate memory: rehash_list.");
-	counter = 0;
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_192[i].LO & (hash_table_size - 1);
+			{
+				if (bt_malloc((void **)&rehash_list, counter * sizeof(unsigned int)))
+					bt_error("Failed to allocate memory: rehash_list.");
+				counter = 0;
+				for (i = 0; i < num_loaded_hashes; i++) {
+					unsigned int idx = loaded_hashes_192[i].LO & (hash_table_size - 1);
 
-		if (collisions[idx] == 3) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
-			}
-			else if (hash_table[idx].iter == 1) {
-				if (check_equal(hash_table[idx].store_loc1, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc2 = i;
-				}
-			}
-			else if (check_equal(hash_table[idx].store_loc1, i) ||
-				 check_equal(hash_table[idx].store_loc2, i))
-				set_zero(i);
-		}
+					if (collisions[idx] == 3) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (hash_table[idx].iter == 1) {
+							if (check_equal(hash_table[idx].store_loc1, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc2 = i;
+							}
+						}
+						else if (check_equal(hash_table[idx].store_loc1, i) ||
+						         check_equal(hash_table[idx].store_loc2, i))
+							set_zero(i);
+					}
 
-		else if (collisions[idx] >= 4) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
-			}
-			else if (hash_table[idx].iter == 1) {
-				if (check_equal(hash_table[idx].store_loc1, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc2 = i;
+					else if (collisions[idx] >= 4) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (hash_table[idx].iter == 1) {
+							if (check_equal(hash_table[idx].store_loc1, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc2 = i;
+							}
+
+						}
+						else if (hash_table[idx].iter == 2) {
+							if (check_equal(hash_table[idx].store_loc1, i) ||
+							    check_equal(hash_table[idx].store_loc2, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc3 = i;
+							}
+						}
+						else if (hash_table[idx].iter >= 3) {
+							if (check_equal(hash_table[idx].store_loc1, i) ||
+							    check_equal(hash_table[idx].store_loc2, i) ||
+							    check_equal(hash_table[idx].store_loc3, i))
+								set_zero(i);
+							else {
+								if (collisions[idx] > 4)
+									rehash_list[counter++] = i;
+							}
+						}
+					}
 				}
 
-			}
-			else if (hash_table[idx].iter == 2) {
-				if (check_equal(hash_table[idx].store_loc1, i) ||
-				    check_equal(hash_table[idx].store_loc2, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc3 = i;
-				}
-			}
-			else if (hash_table[idx].iter >= 3) {
-				if (check_equal(hash_table[idx].store_loc1, i) ||
-				    check_equal(hash_table[idx].store_loc2, i) ||
-				    check_equal(hash_table[idx].store_loc3, i))
-					set_zero(i);
-				else {
-					if (collisions[idx] > 4)
-						rehash_list[counter++] = i;
-				}
+				if (counter)
+					remove_duplicates_final(counter, counter + (counter >> 1), rehash_list);
+				bt_free((void **)&rehash_list);
 			}
 		}
 	}
-
-	if (counter)
-		remove_duplicates_final(counter, counter + (counter >> 1), rehash_list);
-	bt_free((void **)&rehash_list);
-}
-}
-}
 
 #if 0
 	{	unsigned int col1 = 0, col2 = 0, col3 = 0, col4 = 0, col5a = 0;
@@ -444,8 +444,8 @@ unsigned int remove_duplicates_192(unsigned int num_loaded_hashes, unsigned int 
 		col3 *= 3;
 		col4 *= 4;
 		fprintf(stderr, "Statistics:%Lf %Lf %Lf %Lf %Lf\n", (long double)col1 / (long double)num_loaded_hashes,
-		  (long double)col2 / (long double)num_loaded_hashes, (long double)col3 / (long double)num_loaded_hashes,
-			(long double)col4 / (long double)num_loaded_hashes, (long double)col5a / (long double)num_loaded_hashes);
+		        (long double)col2 / (long double)num_loaded_hashes, (long double)col3 / (long double)num_loaded_hashes,
+		        (long double)col4 / (long double)num_loaded_hashes, (long double)col5a / (long double)num_loaded_hashes);
 
 	}
 #endif

--- a/src/bt_hash_type_64.c
+++ b/src/bt_hash_type_64.c
@@ -88,7 +88,7 @@ int test_tables_64(unsigned int num_loaded_hashes, OFFSET_TABLE_WORD *offset_tab
 #if _OPENMP
 #pragma omp for
 #endif
-	for (i = 0; i < num_loaded_hashes; i++) {
+		for (i = 0; i < num_loaded_hashes; i++) {
 			hash = loaded_hashes_64[i];
 			hash_table_idx =
 				calc_ht_idx_64(i,
@@ -177,9 +177,9 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 
 	counter = 0;
 	for (i = 0; i < hash_table_size; i++) {
-		 hash_table[i].collisions = collisions[i];
-		 hash_table[i].iter = 0;
-		 hash_table[i].store_loc1 = hash_table[i].store_loc2 =
+		hash_table[i].collisions = collisions[i];
+		hash_table[i].iter = 0;
+		hash_table[i].store_loc1 = hash_table[i].store_loc2 =
 			hash_table[i].idx_hash_loc_list = 0xffffffff;
 		if (hash_table[i].collisions > 3)
 			hash_table[i].idx_hash_loc_list = counter++;
@@ -221,7 +221,7 @@ static void remove_duplicates_final(unsigned int num_loaded_hashes, unsigned int
 					hash_table[idx].store_loc2 = k;
 			}
 			else if (check_equal(hash_table[idx].store_loc1, k) ||
-				 check_equal(hash_table[idx].store_loc2, k))
+			         check_equal(hash_table[idx].store_loc2, k))
 				set_zero(k);
 		}
 
@@ -281,19 +281,19 @@ unsigned int remove_duplicates_64(unsigned int num_loaded_hashes, unsigned int h
 #if _OPENMP
 #pragma omp parallel private(i)
 #endif
-{
+	{
 #if _OPENMP
 #pragma omp for
 #endif
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_64[i] & (hash_table_size - 1);
+		for (i = 0; i < num_loaded_hashes; i++) {
+			unsigned int idx = loaded_hashes_64[i] & (hash_table_size - 1);
 #if _OPENMP
 #pragma omp atomic
 #endif
-		collisions[idx]++;
-	}
+			collisions[idx]++;
+		}
 
-	counter = 0;
+		counter = 0;
 #if _OPENMP
 #pragma omp barrier
 #endif
@@ -301,108 +301,108 @@ unsigned int remove_duplicates_64(unsigned int num_loaded_hashes, unsigned int h
 #if _OPENMP
 #pragma omp for
 #endif
-	for (i = 0; i < hash_table_size; i++) {
-		hash_table[i].iter = 0;
-		if (collisions[i] > 4)
+		for (i = 0; i < hash_table_size; i++) {
+			hash_table[i].iter = 0;
+			if (collisions[i] > 4)
 #if _OPENMP
 #pragma omp atomic
 #endif
-			 counter += (collisions[i] - 3);
-	}
+				counter += (collisions[i] - 3);
+		}
 #if _OPENMP
 #pragma omp barrier
 
 #pragma omp sections
 #endif
-{
+		{
 #if _OPENMP
 #pragma omp section
 #endif
-{
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_64[i] & (hash_table_size - 1);
+			{
+				for (i = 0; i < num_loaded_hashes; i++) {
+					unsigned int idx = loaded_hashes_64[i] & (hash_table_size - 1);
 
-		if (collisions[idx] == 2) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
+					if (collisions[idx] == 2) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (check_equal(hash_table[idx].store_loc1, i))
+							set_zero(i);
+					}
+				}
 			}
-			else if (check_equal(hash_table[idx].store_loc1, i))
-				set_zero(i);
-		}
-	}
-}
 
 #if _OPENMP
 #pragma omp section
 #endif
-{
-	if (bt_malloc((void **)&rehash_list, counter * sizeof(unsigned int)))
-		bt_error("Failed to allocate memory: rehash_list.");
-	counter = 0;
-	for (i = 0; i < num_loaded_hashes; i++) {
-		unsigned int idx = loaded_hashes_64[i] & (hash_table_size - 1);
+			{
+				if (bt_malloc((void **)&rehash_list, counter * sizeof(unsigned int)))
+					bt_error("Failed to allocate memory: rehash_list.");
+				counter = 0;
+				for (i = 0; i < num_loaded_hashes; i++) {
+					unsigned int idx = loaded_hashes_64[i] & (hash_table_size - 1);
 
-		if (collisions[idx] == 3) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
-			}
-			else if (hash_table[idx].iter == 1) {
-				if (check_equal(hash_table[idx].store_loc1, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc2 = i;
-				}
-			}
-			else if (check_equal(hash_table[idx].store_loc1, i) ||
-				 check_equal(hash_table[idx].store_loc2, i))
-				set_zero(i);
-		}
+					if (collisions[idx] == 3) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (hash_table[idx].iter == 1) {
+							if (check_equal(hash_table[idx].store_loc1, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc2 = i;
+							}
+						}
+						else if (check_equal(hash_table[idx].store_loc1, i) ||
+						         check_equal(hash_table[idx].store_loc2, i))
+							set_zero(i);
+					}
 
-		else if (collisions[idx] >= 4) {
-			if (!hash_table[idx].iter) {
-				hash_table[idx].iter++;
-				hash_table[idx].store_loc1 = i;
-			}
-			else if (hash_table[idx].iter == 1) {
-				if (check_equal(hash_table[idx].store_loc1, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc2 = i;
+					else if (collisions[idx] >= 4) {
+						if (!hash_table[idx].iter) {
+							hash_table[idx].iter++;
+							hash_table[idx].store_loc1 = i;
+						}
+						else if (hash_table[idx].iter == 1) {
+							if (check_equal(hash_table[idx].store_loc1, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc2 = i;
+							}
+
+						}
+						else if (hash_table[idx].iter == 2) {
+							if (check_equal(hash_table[idx].store_loc1, i) ||
+							    check_equal(hash_table[idx].store_loc2, i))
+								set_zero(i);
+							else {
+								hash_table[idx].iter++;
+								hash_table[idx].store_loc3 = i;
+							}
+						}
+						else if (hash_table[idx].iter >= 3) {
+							if (check_equal(hash_table[idx].store_loc1, i) ||
+							    check_equal(hash_table[idx].store_loc2, i) ||
+							    check_equal(hash_table[idx].store_loc3, i))
+								set_zero(i);
+							else {
+								if (collisions[idx] > 4)
+									rehash_list[counter++] = i;
+							}
+						}
+					}
 				}
 
-			}
-			else if (hash_table[idx].iter == 2) {
-				if (check_equal(hash_table[idx].store_loc1, i) ||
-				    check_equal(hash_table[idx].store_loc2, i))
-					set_zero(i);
-				else {
-					hash_table[idx].iter++;
-					hash_table[idx].store_loc3 = i;
-				}
-			}
-			else if (hash_table[idx].iter >= 3) {
-				if (check_equal(hash_table[idx].store_loc1, i) ||
-				    check_equal(hash_table[idx].store_loc2, i) ||
-				    check_equal(hash_table[idx].store_loc3, i))
-					set_zero(i);
-				else {
-					if (collisions[idx] > 4)
-						rehash_list[counter++] = i;
-				}
+				if (counter)
+					remove_duplicates_final(counter, counter + (counter >> 1), rehash_list);
+				bt_free((void **)&rehash_list);
 			}
 		}
 	}
-
-	if (counter)
-		remove_duplicates_final(counter, counter + (counter >> 1), rehash_list);
-	bt_free((void **)&rehash_list);
-}
-}
-}
 
 #if 0
 	{	unsigned int col1 = 0, col2 = 0, col3 = 0, col4 = 0, col5a = 0;
@@ -422,8 +422,8 @@ unsigned int remove_duplicates_64(unsigned int num_loaded_hashes, unsigned int h
 		col3 *= 3;
 		col4 *= 4;
 		fprintf(stderr, "Statistics:%Lf %Lf %Lf %Lf %Lf\n", (long double)col1 / (long double)num_loaded_hashes,
-		  (long double)col2 / (long double)num_loaded_hashes, (long double)col3 / (long double)num_loaded_hashes,
-			(long double)col4 / (long double)num_loaded_hashes, (long double)col5a / (long double)num_loaded_hashes);
+		        (long double)col2 / (long double)num_loaded_hashes, (long double)col3 / (long double)num_loaded_hashes,
+		        (long double)col4 / (long double)num_loaded_hashes, (long double)col5a / (long double)num_loaded_hashes);
 
 	}
 #endif

--- a/src/bt_twister.c
+++ b/src/bt_twister.c
@@ -64,98 +64,97 @@ static mt_uint32   *next;          // next random value is computed from here
 static int      left = -1;      // can *next++ this many times before reloading
 
 void seedMT(mt_uint32 seed)
- {
-    //
-    // We initialize state[0..(N-1)] via the generator
-    //
-    //   x_new = (69069 * x_old) mod 2^32
-    //
-    // from Line 15 of Table 1, p. 106, Sec. 3.3.4 of Knuth's
-    // _The Art of Computer Programming_, Volume 2, 3rd ed.
-    //
-    // Notes (SJC): I do not know what the initial state requirements
-    // of the Mersenne Twister are, but it seems this seeding generator
-    // could be better.  It achieves the maximum period for its modulus
-    // (2^30) iff x_initial is odd (p. 20-21, Sec. 3.2.1.2, Knuth); if
-    // x_initial can be even, you have sequences like 0, 0, 0, ...;
-    // 2^31, 2^31, 2^31, ...; 2^30, 2^30, 2^30, ...; 2^29, 2^29 + 2^31,
-    // 2^29, 2^29 + 2^31, ..., etc. so I force seed to be odd below.
-    //
-    // Even if x_initial is odd, if x_initial is 1 mod 4 then
-    //
-    //   the          lowest bit of x is always 1,
-    //   the  next-to-lowest bit of x is always 0,
-    //   the 2nd-from-lowest bit of x alternates      ... 0 1 0 1 0 1 0 1 ... ,
-    //   the 3rd-from-lowest bit of x 4-cycles        ... 0 1 1 0 0 1 1 0 ... ,
-    //   the 4th-from-lowest bit of x has the 8-cycle ... 0 0 0 1 1 1 1 0 ... ,
-    //    ...
-    //
-    // and if x_initial is 3 mod 4 then
-    //
-    //   the          lowest bit of x is always 1,
-    //   the  next-to-lowest bit of x is always 1,
-    //   the 2nd-from-lowest bit of x alternates      ... 0 1 0 1 0 1 0 1 ... ,
-    //   the 3rd-from-lowest bit of x 4-cycles        ... 0 0 1 1 0 0 1 1 ... ,
-    //   the 4th-from-lowest bit of x has the 8-cycle ... 0 0 1 1 1 1 0 0 ... ,
-    //    ...
-    //
-    // The generator's potency (min. s>=0 with (69069-1)^s = 0 mod 2^32) is
-    // 16, which seems to be alright by p. 25, Sec. 3.2.1.3 of Knuth.  It
-    // also does well in the dimension 2..5 spectral tests, but it could be
-    // better in dimension 6 (Line 15, Table 1, p. 106, Sec. 3.3.4, Knuth).
-    //
-    // Note that the random number user does not see the values generated
-    // here directly since reloadMT() will always munge them first, so maybe
-    // none of all of this matters.  In fact, the seed values made here could
-    // even be extra-special desirable if the Mersenne Twister theory says
-    // so-- that's why the only change I made is to restrict to odd seeds.
-    //
+{
+	//
+	// We initialize state[0..(N-1)] via the generator
+	//
+	//   x_new = (69069 * x_old) mod 2^32
+	//
+	// from Line 15 of Table 1, p. 106, Sec. 3.3.4 of Knuth's
+	// _The Art of Computer Programming_, Volume 2, 3rd ed.
+	//
+	// Notes (SJC): I do not know what the initial state requirements
+	// of the Mersenne Twister are, but it seems this seeding generator
+	// could be better.  It achieves the maximum period for its modulus
+	// (2^30) iff x_initial is odd (p. 20-21, Sec. 3.2.1.2, Knuth); if
+	// x_initial can be even, you have sequences like 0, 0, 0, ...;
+	// 2^31, 2^31, 2^31, ...; 2^30, 2^30, 2^30, ...; 2^29, 2^29 + 2^31,
+	// 2^29, 2^29 + 2^31, ..., etc. so I force seed to be odd below.
+	//
+	// Even if x_initial is odd, if x_initial is 1 mod 4 then
+	//
+	//   the          lowest bit of x is always 1,
+	//   the  next-to-lowest bit of x is always 0,
+	//   the 2nd-from-lowest bit of x alternates      ... 0 1 0 1 0 1 0 1 ... ,
+	//   the 3rd-from-lowest bit of x 4-cycles        ... 0 1 1 0 0 1 1 0 ... ,
+	//   the 4th-from-lowest bit of x has the 8-cycle ... 0 0 0 1 1 1 1 0 ... ,
+	//    ...
+	//
+	// and if x_initial is 3 mod 4 then
+	//
+	//   the          lowest bit of x is always 1,
+	//   the  next-to-lowest bit of x is always 1,
+	//   the 2nd-from-lowest bit of x alternates      ... 0 1 0 1 0 1 0 1 ... ,
+	//   the 3rd-from-lowest bit of x 4-cycles        ... 0 0 1 1 0 0 1 1 ... ,
+	//   the 4th-from-lowest bit of x has the 8-cycle ... 0 0 1 1 1 1 0 0 ... ,
+	//    ...
+	//
+	// The generator's potency (min. s>=0 with (69069-1)^s = 0 mod 2^32) is
+	// 16, which seems to be alright by p. 25, Sec. 3.2.1.3 of Knuth.  It
+	// also does well in the dimension 2..5 spectral tests, but it could be
+	// better in dimension 6 (Line 15, Table 1, p. 106, Sec. 3.3.4, Knuth).
+	//
+	// Note that the random number user does not see the values generated
+	// here directly since reloadMT() will always munge them first, so maybe
+	// none of all of this matters.  In fact, the seed values made here could
+	// even be extra-special desirable if the Mersenne Twister theory says
+	// so-- that's why the only change I made is to restrict to odd seeds.
+	//
 
-    register mt_uint32 x = (seed | 1U) & 0xFFFFFFFFU, *s = state;
-    register int    j;
+	register mt_uint32 x = (seed | 1U) & 0xFFFFFFFFU, *s = state;
+	register int    j;
 
-    for (left=0, *s++=x, j=N; --j;
-        *s++ = (x*=69069U) & 0xFFFFFFFFU);
- }
+	for (left=0, *s++=x, j=N; --j;
+	     *s++ = (x*=69069U) & 0xFFFFFFFFU);
+}
 
 
 static mt_uint32 reloadMT(void)
- {
-    register mt_uint32 *p0=state, *p2=state+2, *pM=state+M, s0, s1;
-    register int    j;
+{
+	register mt_uint32 *p0=state, *p2=state+2, *pM=state+M, s0, s1;
+	register int    j;
 
-    if (left < -1)
-        seedMT(4357U);
+	if (left < -1)
+		seedMT(4357U);
 
-    left=N-1, next=state+1;
+	left=N-1, next=state+1;
 
-    for (s0=state[0], s1=state[1], j=N-M+1; --j; s0=s1, s1=*p2++)
-        *p0++ = *pM++ ^ (mixBits(s0, s1) >> 1) ^ (loBit(s1) ? K : 0U);
+	for (s0=state[0], s1=state[1], j=N-M+1; --j; s0=s1, s1=*p2++)
+		*p0++ = *pM++ ^ (mixBits(s0, s1) >> 1) ^ (loBit(s1) ? K : 0U);
 
-    for (pM=state, j=M; --j; s0=s1, s1=*p2++)
-        *p0++ = *pM++ ^ (mixBits(s0, s1) >> 1) ^ (loBit(s1) ? K : 0U);
+	for (pM=state, j=M; --j; s0=s1, s1=*p2++)
+		*p0++ = *pM++ ^ (mixBits(s0, s1) >> 1) ^ (loBit(s1) ? K : 0U);
 
-    s1=state[0], *p0 = *pM ^ (mixBits(s0, s1) >> 1) ^ (loBit(s1) ? K : 0U);
-    s1 ^= (s1 >> 11);
-    s1 ^= (s1 <<  7) & 0x9D2C5680U;
-    s1 ^= (s1 << 15) & 0xEFC60000U;
-    return(s1 ^ (s1 >> 18));
- }
+	s1=state[0], *p0 = *pM ^ (mixBits(s0, s1) >> 1) ^ (loBit(s1) ? K : 0U);
+	s1 ^= (s1 >> 11);
+	s1 ^= (s1 <<  7) & 0x9D2C5680U;
+	s1 ^= (s1 << 15) & 0xEFC60000U;
+	return(s1 ^ (s1 >> 18));
+}
 
 
 mt_uint32 randomMT(void)
- {
-    mt_uint32 y;
+{
+	mt_uint32 y;
 
-    if (--left < 0)
-        return(reloadMT());
+	if (--left < 0)
+		return(reloadMT());
 
-    y  = *next++;
-    y ^= (y >> 11);
-    y ^= (y <<  7) & 0x9D2C5680U;
-    y ^= (y << 15) & 0xEFC60000U;
-    return(y ^ (y >> 18));
- }
+	y  = *next++;
+	y ^= (y >> 11);
+	y ^= (y <<  7) & 0x9D2C5680U;
+	y ^= (y << 15) & 0xEFC60000U;
+	return(y ^ (y >> 18));
+}
 
 #endif
-

